incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,12 @@
+//////////  S V G  M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+@mixin svg($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
+    background:
+        repeat: no-repeat
+        size: contain
+    width: $width
+    height: $height
+    display: inline-block
```

Comment:
> Our icon-font mixin has `display: inline-block` too.

That's understandable due to the majority use of icons. SVG image elements, however, can be anything; they're not always going to be inline icons.

> If it turns out not to be useful we can always take it out later. Sometimes you just have to see what happens, you can't future proof everything :D

I prefer libraries to do only as much as is necessary. How about creating `@mixin svg-icon` that calls `@include svg` and adds `display: inline-block`? That will then allow us to target svg icons with more specific things when we come to build an svg icon system to replace the current icon font system.